modules/auxiliary/dos/windows/http/rb_dos_iis_2022_21907.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+###
+#
+# This module performs a DOS attack using a simple HTTP request.
+#
+###
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Auxiliary::Dos
+
+  ##
+  # Initialize informations about module.
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'CVE-2022-21907: HTTP Protocol Stack Remote Code Execution Vulnerability - Windows IIS DOS BlueScreen',
+        'Description' => 'This module can be used to perform a DOS attack on IIS server. This module exploit the CVE-2022-21907 and causes a Blue Screen with only one payload.',
+        'License' => MSF_LICENSE,
+        'Author' => ['Maurice LAMBERT <mauricelambert434@gmail.com>'],
+        'Platform' => 'win',
+        'References' => [
+          %w[CVE 2022-21907],

```

Comment:
For consistency with the rest of the references

```suggestion
          ['CVE', '2022-21907'],

```